src/components/Certify/Certify.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <v-card flat id="AR-step-4-container">
+    <div class="certify-container">
+      <div class="certifiedby-container">
+        <label>Legal Name</label>
+        <div class="value certifiedby">
+          <v-text-field
+            filled
+            persistent-hint
+            id="certified-by-textfield"
+            label="Person's legal name"
+            hint="Legal name of authorized person"
+            :value="certifiedBy"
+            :rules="[(v) => !!v || 'A person\'s legal name is required.']"
+            @input="emitCertifiedBy($event)"
+          />
+        </div>
+      </div>
+      <v-checkbox
+        :value="isCertified"
+        @change="emitIsCertified($event)"
+        id="isCertified-checkbox"
+      >
+        <template slot="label">
+          <div class="certify-stmt">
+            I,
+            <strong>{{ trimmedCertifiedBy || "[Legal Name]" }}</strong>
+            , certify that I have relevant knowledge of the
+            {{ entityDisplay || "association" }} and that I am authorized to
+            make this filing.
```

Comment:
Again here, please check the required wording for business-edit-ui.

Reply:
This is still not in line with the design. Please see https://preview.uxpin.com/c53183da94b200338185d9557b11dd9a860f9092#/pages/131426862/simulate/no-panels